<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sample_layout_1Class</class>
 <widget class="QMainWindow" name="sample_layout_1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sample_layout_1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="debugOut">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startCamera">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Camera</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>731</width>
      <height>111</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>MIDDLE_BASE</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>CENTER</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>WRIST</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>g_depthData[x][y]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>g_depthData[x][y] mapped</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>461</width>
      <height>85</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>To Palm</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>To Finger</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>O</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>θ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>θ°</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>235</red>
       <green>255</green>
       <blue>254</blue>
      </color>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="stopCamera">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameCount">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="QGV_FlexationDial_Left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>198</width>
      <height>104</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_HandStatus_Left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>198</width>
      <height>115</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Hand Detected</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Hand Tracked</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Hand Calibrated</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="QGV_FlexationDial_Right">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>198</width>
      <height>104</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_HandStatus_Right">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>198</width>
      <height>115</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Hand Detected</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Hand Tracked</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Hand Calibrated</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="QGV_Video_depth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="QGV_Video_color">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnX">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>btnX</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QTextEdit" name="te_paybackFile">
       <property name="maximumSize">
        <size>
         <width>16777214</width>
         <height>27</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;example.rssdk&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_File">
       <property name="text">
        <string>File: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QTextEdit" name="te_recordFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;example.rssdk&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_record">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_playback">
       <property name="text">
        <string>Playback</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_file2">
       <property name="text">
        <string>File: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Stream">
       <property name="text">
        <string>Stream</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radio_playback">
       <property name="text">
        <string/>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stream_type</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radio_record">
       <property name="text">
        <string/>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stream_type</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radio_stream">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">stream_type</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QSpinBox" name="sb_IVCAMAccuracy">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_default_IVCAMAccuracy">
       <property name="text">
        <string>Reset Default (xxx)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IVCAMLaserPower</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sb_IVCAMLaserPower"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_default_IVCAMLaserPower">
       <property name="text">
        <string>Reset Default (xxx)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>IVCAMFilter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sb_IVCAMFilter"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pb_default_IVCAMFilter">
       <property name="text">
        <string>Reset Default (xxx)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>IVCAMMotionRangeTradeOff</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sb_IVCAMMotionRangeTradeOff">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pb_default_IVCAMMotionRangeTradeOff">
       <property name="text">
        <string>Reset Default (xxx)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QPushButton" name="pb_default_all">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IVCAMAccuracy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>debugOut</zorder>
   <zorder>startCamera</zorder>
   <zorder>tableWidget</zorder>
   <zorder>tableWidget_2</zorder>
   <zorder>stopCamera</zorder>
   <zorder>frameCount</zorder>
   <zorder>QGV_FlexationDial_Left</zorder>
   <zorder>table_HandStatus_Left</zorder>
   <zorder>QGV_FlexationDial_Right</zorder>
   <zorder>table_HandStatus_Right</zorder>
   <zorder>QGV_Video_depth</zorder>
   <zorder>QGV_Video_color</zorder>
   <zorder>btnX</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sample_layout_1.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="stream_type"/>
 </buttongroups>
</ui>
